account_invoice_provider_reference/README.rst

```diff
@@ -0,0 +1,58 @@
+================================
+Partner Structured Communication
```

Comment:
maybe this is no more accurate related to the name of the module and your input comment for this PR